certbot/cli.py

```diff
@@ -991,10 +991,18 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         default=flag_default("tls_sni_01_port"),
         help=config_help("tls_sni_01_port"))
     helpful.add(
+        ["testing", "standalone", "apache", "nginx"], "--tls-sni-01-addr",
```

Comment:
This line seems to indicate that the `--tls-sni-01-addr` argument is applicable to the standalone, apache, and nginx plugins. However, this CL only makes use of it in the standalone plugin (and that seems to be intentional).